tools/util.wdl

```diff
@@ -68,3 +68,44 @@ task prepare_read_groups_for_star {
         read_groups: "The read group portion of a BAM header as a string"
     }
 }
+
+task file_basename {
```

Comment:
I almost feel like this isn't worth making its own task. I didn't look where it is used, but it it possible to just call this inline with the other commands in tasks where it is needed?